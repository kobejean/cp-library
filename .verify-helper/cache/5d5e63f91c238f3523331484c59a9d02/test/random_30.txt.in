98
106090051 66120097 293222344
558899253 523104252 305938835
45517267 36741606 344441303
676146543 6207441 525125233
81946219 23503847 696775785
694053035 353727995 828859818
528669277 400160443 847753316
705065779 626518221 537586024
978852530 579806065 10308910
682958738 270494790 852007658
470923192 428734928 432827454
344358396 243254915 2623538
360699167 135198905 554959365
812031902 548859596 791354898
230345990 45414911 965203476
110991466 45250862 966973820
490422458 418169966 187117484
62967032 61427647 907793624
75486553 21050844 377432765
533702502 337458735 533702502
208933643 27091092 417867286
491910454 201701893 983820908
444142253 224222235 888284506
726401894 342270823 726401894
735352327 392721414 735352327
292693924 48189973 585387848
894047477 225793745 894047477
718636504 149357800 718636504
300944561 56401450 601889122
113522269 28650695 113522269
356454463 53242131 712908926
314682883 197309930 629365766
420982097 218804485 420982097
550575648 59049527 550575648
329395531 243400498 988186593
198104715 78488288 396209430
904386181 888481010 904386181
49439660 30650679 247198300
308958319 116901419 617916638
949467010 679701044 949467010
394959361 362163618 394959361
860901104 63104042 860901104
769385537 449817545 769385537
524894176 486142661 524894176
616240777 61933632 616240777
411607888 297936401 411607888
882907829 694225839 882907829
103407047 77797997 930663423
227569387 218626018 910277548
130249807 109347416 260499614
964527257 598524301 964527257
121631345 115906458 608156725
35747521 26032904 142990084
399435748 153051276 399435748
929799463 556391940 929799463
820335369 490304563 820335369
777531067 438376205 777531067
900933464 7804593 900933464
51464737 23973723 102929474
769540470 481360988 769540470
683906689 60454345 683906689
700964992 510149268 700964992
557073809 123720407 557073809
442501071 3652516 885002142
769972843 467441494 769972843
385268138 211689490 770536276
878850451 377012126 878850451
716336115 695626680 716336115
999999924 883116816 337662312
999999982 526315780 842105248
999999975 51282050 948717925
999999968 249999992 295454536
999999984 790322568 822580632
999999984 387096768 924731168
999999993 647058819 705882348
999999962 885245868 245901630
999999980 782608680 869565200
999999936 555555520 652777736
999999987 809523799 761904752
999999963 649122783 228070167
999999990 484848480 515151510
999999944 26315788 65789470
1000000000 750000000 250000000
999999990 931506840 712328760
999999993 372549017 254901959
999999976 470588224 147058820
999999910 76923070 956043870
999999968 329545444 11363636
999999968 22727272 170454540
1000000000 500000000 750000000
999999968 977272696 602272708
999999988 192307690 173076921
999999916 162790684 151162778
999999996 416666665 83333333
999999936 179487168 910256352
999999962 262295072 16393442
999999984 870967728 483870960
999999980 315217385 184782605
