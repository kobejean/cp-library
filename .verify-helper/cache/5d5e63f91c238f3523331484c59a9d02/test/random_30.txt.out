4028554
33
45517255
608835279
75
694052950
98564767
29
-1
59
235461496
-1
94
406015899
-1
57
245211156
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
27
16
1
33
27
45
9
33
6
16
16
41
1
30
1
38
26
24
70
59
82
2
10
22
32
7
2
45
23
47
